px4_offboard/offboard_control.py

```diff
@@ -68,16 +68,18 @@ def __init__(self):
         self.timer = self.create_timer(timer_period, self.cmdloop_callback)
 
         self.nav_state = VehicleStatus.NAVIGATION_STATE_MAX
+        self.arming_state = VehicleStatus.ARMING_STATE_DISARMED
         self.dt = timer_period
         self.theta = 0.0
-        self.radius = 10.0
+        self.radius = 1.0
```

Comment:
Could we maybe make this a parameter, so that it can be configured from the hardware launchfile?

Reply:
That works, I can tidy that up on another push.

Reply:
Just added this! Note that there aren't any parameter callbacks because changing these live could result in discontinuities - I can still add them if need be, but would rather guard against that by not having these callbacks and only have them at time of launch. How does that sound?